### PR TITLE
Add com.warlordsoftwares.media-downloader

### PR DESCRIPTION
**Pull request by rishabh3354**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by rishabh3354** [comment]
bot, build com.warlordsoftwares.media-downloader

**Comment by rishabh3354** [comment]
bot, build com.warlordsoftwares.media-downloader

**Comment by rishabh3354** [comment]
bot, build com.warlordsoftwares.media-downloader

**Comment by rishabh3354** [comment]
bot, build com.warlordsoftwares.media-downloader

File: com.warlordsoftwares.media-downloader.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - BASEAPP_REMOVE_WEBENGINE=1
+command: youtube-dl-pro.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfsd
```
Comment (bbhtt) [inline com.warlordsoftwares.media-downloader.yml:20]: You are setting `screenshot-directory=~/Pictures/UMD4-MPV-screenshots` as the screenshot directory, so I assume mpv requires fs access for that path?

Comment (rishabh3354) [reply com.warlordsoftwares.media-downloader.yml:20]: Actually i dont need to change the default path for screenshot! Thanks good point! +1

**Comment by bbhtt** [comment]
Desktop file shows the old name

image

**Comment by rishabh3354** [comment]
name changed from desktop file also

**Comment by rishabh3354** [comment]
bot, build com.warlordsoftwares.media-downloader

**Comment by rishabh3354** [comment]
> 

done

**Pull request closed without merging**